tasks/win_agent_version.yml

```diff
@@ -0,0 +1,4 @@
+---
+- name: set agent download filename to a specific version
+  set_fact:
+    dd_download_url: "https://s3.amazonaws.com/ddagent-windows-stable/ddagent-cli-{{ datadog_agent_version }}.msi"
```

Comment:
Do you think moving the URL to the `defaults/main.yml` so customers can change it make sens ? Do we have users that cache our msi ?

Reply:
Good question. I thought about it, as it would help me for testing, but it didn't seem generic enough for customer use... but I'm willing to be convinced :). Note that in the specific version case it's computed.

Reply:
I think it doesn't cost us anything to move the `https://s3.amazonaws.com/ddagent-windows-stable/` part of the URL to the default (for both latest and a specific version). worst case it doesn't change anything, best case it gives user more flexibility.